src/emoji-picker/components/blockedEmojis.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+export const blockedEmoji = [
+  'beer',
+  'beers',
+  'wine_glass',
+  'cocktail',
+  'tropical_drink',
+  'tumbler_glass',
+  'smoking',
+  'middle_finger',
+  'wink',
+  'stuck_out_tongue_winking_eye',
+  'kissing_heart',
+  'kissing',
+  'kissing_closed_eyes',
+  'kissing_smiling_eyes',
+  'tongue',
+  'eggplant',
+  'peach',
+  'dancers',
+  'men-with-bunny-ears-partying',
+  'women-with-bunny-ears-partying',
+  'syringe',
+  'pill',
+];
```

Comment:
The blocked emoji list is per helpline. For now, only KHP is requesting this. I think we shouldn't make all these emojis blocked for all helplines. 

My suggestion is that we have this list on the helpline config file (`ca-staging.ts` and possibly `as-development.ts`)